xoinvader/ingame.py

```diff
@@ -93,13 +93,24 @@ def handle(self):
         # Some other event logic
 
 
-class TestWave(EnemyWave):
+class TestLevel(Level):
     def __init__(self, state_object_adder, speed):
-        super(TestWave, self).__init__(speed)
-        self._state_object_adder = state_object_adder
+        super(TestLevel, self).__init__(speed)
+        self._state_add = state_object_adder
+
+        self._player_ship = Playership(
+            Settings.layout.field.player, Settings.layout.field.edge)
+        self._state_add(self._player_ship)
+
         self.add_event(1, self.spawn4)
         self.add_event(100, lambda: None)
 
+        self.bg = Background(Settings.path.level1bg)
+        self.bg.start(True)
+        self.bg.speed = 10
```

Comment:
Nevertheless I want to move settings speed and looping to background constructor. You can leave getters/setters for runtime changes, but let's use constructors to _construct_ object we want. Imagine weapon or ship creation like this:
```python
s = PlayerShip(s_pos)
s.dx = 1
s.animation = Animation(...)
...
...
...
```